src/nvim/option.c

```diff
@@ -1958,8 +1958,8 @@ static char_u *illegal_char(char_u *errbuf, size_t errbuflen, int c)
   if (errbuf == NULL) {
     return (char_u *)"";
   }
-  snprintf((char *)errbuf, errbuflen, _("E539: Illegal character <%s>"),
-           (char *)transchar(c));
+  vim_snprintf((char *)errbuf, errbuflen, _("E539: Illegal character <%s>"),
+               (char *)transchar(c));
```

Comment:
what was the reason for this change?

Reply:
I would use `snprintf` but used this for consistency since I found it already being used like here:
https://github.com/neovim/neovim/blob/dfb7f6b3492edfd60583f76ee064ac76b1b913d8/src/nvim/option.c#L4405-L4406